src/components/PlatformNavigator/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState, useEffect } from 'react';
+import { Button, Link, Flex, Text } from '@aws-amplify/ui-react';
+import { IconChevron } from '@/components/Icons';
+import { frameworks } from '@/constants/frameworks';
+import { InfoPopover } from './InfoPopover';
+import { parseLocalStorage } from '@/utils/parseLocalStorage';
+
+export function PlatformNavigator({ currentPlatform }) {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [showInfo, setShowInfo] = useState<boolean>(false);
+
+  const platformItem = frameworks.filter((platform) => {
+    return platform.title === currentPlatform;
+  })[0];
+
+  useEffect(()=>{
+    let returnVisitor = parseLocalStorage('returnVisitor');
```

Comment:
We only want the info popup to show for first time visitors, it was suggested that we use the marketing cookie instead of localStorage for that but I decided to use localStorage because then we can control when and how this value is set vs relying on the marketing script which we don't control.

Reply:
For me when I visit the staging site I only see the "Info" text, is the popover supposed to be shown too?